Zh3r0_CTF_V2_2021-import_numpy_as_MT/description.yml

```diff
@@ -0,0 +1,8 @@
+name: import numpy as MT
+author: deuterium
+original_ctf: Zh3r0 CTF V2
+year: 2021
+description: >-
+  "Python is so slow! Lets use nUmPy tO MAkE iT FaSTer. Only if there was a module for crypto in it :("
+base64_flag: 'emgzcjB7d2gwX3RoMHVnaDdfN2gzX20zcjUzbm4zXzd3MTU3M3JfdzQ1XzVvX3ByM2QxYzc0YmwzP2MzcnQ0MW5seV9uMHRfbTQ3NDU0bjB9jj'
```

Comment:
Vim user spotted, looks like "jj" made it's way to the end of the flag here :D

It should be 'emgzcjB7d2gwX3RoMHVnaDdfN2gzX20zcjUzbm4zXzd3MTU3M3JfdzQ1XzVvX3ByM2QxYzc0YmwzP2MzcnQ0MW5seV9uMHRfbTQ3NDU0bjB9'

Reply:
> Hey deuterium, these challenges look great.
> 
> I've pointed out some build errors above
> 
> Also instead of importing `flag` from `secret.py`, you can refer to flag inside the challenges with `os.environ["FLAG"]`, this reduces redundancy and ensures the actual flag is always correct.
> 
> Once these are fixed it is ready to merge.

I think I wasnt getting copy errors on my docker (big oof). I would use env and remove those secret.py files, 
Thanks for your suggestions